### DISCUSSION TITLE
rolls

### DISCUSSION THREAD

**Discussion by blotterchains** [Godot-ground/tower-defense#2] [General]
hello Godoers

Please state your role in this discussion

**Comment by danialabdl** [comment]
hello
game desginer
helping in programming if needed

**Comment by sinakhatib** [comment]
hello
game designer
concept artist
artist
animator

**Comment by Soroosh83F** [comment]
Character designer
Composer

**Comment by Alinj7** [comment]
scenario writer
Thinker
